aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -361,99 +371,31 @@ pub fn extension_error(message: impl ToString) -> PartialVMError {
 // ================================= Tests =================================
 
 #[cfg(test)]
-mod test {
+pub mod test {
```

Comment:
why pub this?

Reply:
Forgot to remove that after some refactoring